po/en.po

```diff
@@ -1778,8 +1778,10 @@ msgstr "Illegal file descriptor in redirection “%ls”"
 
 #: parse_constants.h:213
 #, c-format
-msgid "No matches for wildcard '%ls'."
-msgstr "No matches for wildcard “%ls”."
+msgid "No matches for wildcard pattern '%ls'.\n"
+"(Tip: To avoid wildcarding, quote it. To allow no matches, expand it to a variable.)"
```

Comment:
"expand it to a variable" may not be clear enough. Something like "assign the glob to a variable" may be better.